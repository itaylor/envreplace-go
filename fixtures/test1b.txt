#FOO#
The below are not variables per the regex from test1.json
#foo #FOO # BAR# #BAR
