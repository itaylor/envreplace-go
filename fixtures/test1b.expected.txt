test1
The below are not variables per the regex from test1.json
#foo #FOO # BAR# #BAR
